flowstate_interfaces/action/StartProcess.action

```diff
@@ -27,5 +27,8 @@ string result
 ---
 
 # Feedback
-# todo(Yadunund)
+bool ok
+
 string message
+
+string error_message
```

Comment:
```suggestion
bool done

string error_message
```

Would it make sense to call this `done` instead of `ok` to match the corresponding field in the `current_operation_` which is a [google.longrunning.Operation](https://cloud.google.com/service-infrastructure/docs/service-management/reference/rpc/google.longrunning#operation)?

`done` would be different from `success` in the result as the executive may be "done" but the operation did not succeed.

And perhaps we can have a single `error_message` field since this is only a feedback? Or do you think we need separate fields for `message` and `error_message`?